student.rb

```diff
@@ -0,0 +1,18 @@
+# Student class represents a student with classroom information.
+class Student < Person
+  def initialize(id, age, parent_permission: true, classroom: nil)
+    super(id, age, parent_permission)
+    @classroom = classroom
+  end
+
+  def classroom=(classroom)
+    attr_reader :classroom
```

Comment:
- [ ] Nice work making the changes suggested by the previous reviewer :clap: However, The instance variable get (`attr_reader :classroom`) should be outside of the `classroom` method, it's recommended to add it before the `initialize` method. Kindly remove it from the `classroom` method and add it to the class `Student`
```ruby
class Student < Person
  attr_reader :classroom
  def initialize(id, age, parent_permission: true, classroom: nil)
    super(id, age, parent_permission)
    @classroom = classroom
  end
...
end
```

Reply:
setters moved outside the method